docs/flash_messages.md

```diff
@@ -0,0 +1,26 @@
+# Flash Messages
+
+Flash messages in this application are handled by the FlashHelper module and the FlashMessagesComponent, FlashToastWrapperComponent and FlashToastComponent classes.
```

Comment:
we don't need to be this technical. the main purpose of this document is to show the capabilities of what we have built with flash messages. 

show screenshots of two type of flash messages we have. the usual one and toast. 

how they would use both of them. the possible params they can pass in etc. the same code is good that we have. we just need to make the doc less specific in the places we don't need it. 

the place we do need to be specific is we need to have a section where we say how they can customise the code. both both the normal flash and the toasts. in there say and link to files and parts of code where we are doing the HTML styling and the logic.